docs/adr/0003-graphql-layer-between-backend-frontend.md

```diff
@@ -2,42 +2,36 @@
 
 Date: 2020-04-30
 
-## Related tickets (not available for public)
+## Status
 
-SIG-2568
+**Accepted**
 
-## Disclaimer
+## Related tickets (not available for public)
 
-Since this proposal is quite big the original author (jpoppe) has taken the liberty make this
-proposal a bit more opiniated then it normally should be. (One of the reasons is that the original
-author has done this implementation and fine tuned it a couple of times)
+SIG-2568
 
 ## Context
 
 Since our current backend only serves data through REST and there is no possibility to generate
-generate types/interfaces there is no automated way to validate data on the frontend which is coming
-from the backend.
+types and or interfaces, there is no also no automated way to validate data on the frontend
+which is coming from the backend.
 
-Besides of that there are too many inconstencies between different backend calls and the current
+Besides of that there are too many inconstensies between different backend calls and the current
 backend API documentation is not always correct and/or up to date because it's written manually
 and not generated from code.
 
-Because of this our `React components` are over complicated since we need to mangle data all the
-time to be able to send a POST/PUT request from data which got from a GET request.
+Because of this our `React components` are overcomplicated since we need to transform thr API output to a
```

Comment:
'the' instead of 'thr'

Reply:
Thanks, will enable spell checking on my MarkDown documents so it won't happen anymore ;)